app/javascript/packs/leafletMap.js

```diff
@@ -49,7 +50,7 @@ export async function setupMap() {
 
   L.tileLayer("http://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png", {
     attribution:
-      '&copy; <a href="https://www.openstreetmap.org/copyright">OpenStreetMap</a>',
+      "&copy; <a href='https://www.openstreetmap.org/copyright'>OpenStreetMap</a>",
```

Comment:
Any particular reason for this change?

Reply:
consistency to use "..." instead of '...'